Code/Mods/UI/Camera.cs

```diff
@@ -285,9 +299,44 @@ private static void Character_SetZoomMode_Post(Character __instance, ref bool _z
         if (!_perPlayerSettings[player.ID]._toggle)
             return;
         #endregion
-
+        
         __instance.CharacterCamera.ZoomSensModifier = 1f;
```

Comment:
Perhaps fix this magic value with a predefined const?

Reply:
That would improve readibility but I don't know where Vheos got that magic number which makes it bit hard to name it.

Reply:
Ok doing research, its apparent that vanilla sensitivity is 1/4 of this magic number. Do we want to reset it to vanilla, unhardcode, or constant it to its current value. The variable that currently contains sensitivity is a vector3 so it would have to be changed to vector4 and the code needs to be fixed up. We could also just add a float would be simpler.